WordPressKit.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name          = "WordPressKit"
-  s.version       = "4.31.0"
+  s.version       = "4.32.0-beta.2"
```

Comment:
When I saw this is jumping from `4.31.0` to `4.32.0-beta.2`, I realized this wasn't updated in https://github.com/wordpress-mobile/WordPressKit-iOS/pull/381.
Since I made a release for `4.32.0-beta.2`, I'm going to update this on develop and edit the release, otherwise WPiOS might run into issues.

Reply:
On second thought, it's probably better to let this be and just wait to this new beta is merged and WPiOS is updated.